openedx/core/djangoapps/edx_global_analytics/README.md

```diff
@@ -0,0 +1,88 @@
+
+## The way sent data looks like on Acceptor
+
+![acceptor_world_map](https://user-images.githubusercontent.com/22666467/27970248-d015fd9c-6356-11e7-906c-8c052cd65b08.png)
+
+## Types of statistics sent, and dependence on statistics level
+
+Firstly, `statistic level` is a string constant in settings (`lms.env.json`) which regulates size of bunch platform sends.
+
+First `Paranoid` level with paranoid constant allows platform to transfer:
+1. Access token for Acceptor server.
+2. Active students amount per last calendar day, week and month*.
+3. Courses amount.
+
+Second `Enthusiast` level and the last one with `enthusiast` constant extends `Paranoid` level with:
+1. Platform URL and platform name.
+2. Platform's latitude and longitude.
+3. Active students per country accordance for last calendar day.
+
+\*active student is a student whose last login datetime value is included in calendar duration.
+
+## Settings
+
+Platform's administrator is able to configure `EdX global analytics` settings in `lms.env.json`.
+This settings already exist after default platform installation, admin just needs to change it.
+
+Settings context:
+```
+...,
+"OPENEDX_LEARNERS_GLOBAL_ANALYTICS": {
+    "CELERY_TIMEZONE": "Europe/Kiev",
+    "OLGA_ACCEPTOR_PERIODIC_TASK_URL": "",
+    "OLGA_ACCEPTOR_PERIODIC_TASK_URL_LOCAL_DEV": "http://192.168.1.10:7000",
+    "PLATFORM_CITY_NAME": "Kiev",
+    "STATISTICS_LEVEL": "enthusiast"
+}, ...
+```
+
+`CELERY_TIMEZONE`: Time zone in edx-platform is enabled by default, platform's administrator can set time zone in `lms.env.json` for `TIME_ZONE` key. `EdX global analytics` supports own time zone setting — time zone platform will fetch and send statistics to Acceptor.
+
+`OLGA_ACCEPTOR_PERIODIC_TASK_URL` and `OLGA_ACCEPTOR_PERIODIC_TASK_URL_LOCAL_DEV` are URLs that platform transfer statistics to. It depends on platform status: development or production.
+
+`PLATFORM_CITY_NAME` — set city name platform located in for gathering latitude and longitude.
+It is required for `enthusiast` level.
+
+`STATISTICS_LEVEL` — direct statistics level (explained above).
+
+## Tests
+
+You are able to run unit tests from `edx-platform` directory with command:
+
+```
+paver test_system -t openedx/core/djangoapps/edx_global_analytics/tests
+```
+
+## Development details
+
+Celery settings located in `lms/envs/aws.py`:
+```
+For scheduling tasks, entries can be added to this dict.
+CELERYBEAT_SCHEDULE = {
+    """
+    'collect_stats' is the celery periodic task that gathers information about the
+    students amount, geographical coordinates of the platform, courses amount and
+    makes a POST request with the data to the appropriate service.
+    """
+
+    'collect_stats': {
+        'task': 'openedx.core.djangoapps.edx_global_analytics.tasks.collect_stats',
+        'schedule': crontab(hour=0, minute=random.randint(1, 59)),
+    }
+}
+```
+
+Run celery task with command:
+
```

Comment:
Add block "architecture" where shortly explain architecture of the project, for example, that it is use celery, now authorizes and how many requests per days are sent.